### DISCUSSION TITLE
Site Name in  Original Path

### DISCUSSION THREAD

**Discussion by aavi92** [microsoft-search/pnp-modern-search#2597] [Q&A] [answered]
Hi, 

 "OriginalPath/Path" column value results are displayed as "siteurl/DocumentLibrary/FileName" is there a way we can change to "SiteName/DocumentLibrary/FileName" ? 

Thanks.

**Comment by wobba** [comment, marked as answer]
Use these managed properties: `SiteTitle` then take the last part of `ParentLink` when split on `/` and use `Filename` at the end.
